scss/_patterns_grid.scss

```diff
@@ -19,13 +19,193 @@ $shelves-column-name: $grid-col-name;
   }
 
   // Demo helper
-  [grid-demo] [class*='col-'] {
+  [grid-demo] [class*='#{$grid-col-name}-'] {
     background: $color-mid-light;
     margin-bottom: $spv-inner--small;
   }
 
-  [grid-outline] [class*='col-'] {
+  [grid-outline] [class*='#{$grid-col-name}-'] {
     outline: 1px solid $color-x-light;
     padding: $spv-inner--small $sph-inner--small;
   }
 }
+
+@mixin vf-p-css-grid {
+  %row-reset {
+    margin-left: 0;
+    margin-right: 0;
+    max-width: $grid-max-width;
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  %span-full-grid-on-mobile {
+    @media (max-width: $breakpoint-medium) {
+      grid-column: 1 / span $grid-columns-mobile;
+    }
+  }
+
+  %span-full-grid-on-tablet {
+    @media (min-width: $breakpoint-medium) and (max-width: $breakpoint-large) {
+      grid-column: 1 / span $grid-columns-tablet;
+    }
+  }
+
+  %span-full-grid-on-desktop {
+    @media (min-width: $breakpoint-large) {
+      grid-column: 1 / span $grid-columns;
+    }
+  }
+
+  [css-grid-demo] [class*='#{$grid-column-prefix}'] {
```

Comment:
The example markup are to be used as example usage. Therefore we need to find another way to style them or then bespoke attrs which are in the packaged version of Vanilla.